gateway/gateway.go

```diff
@@ -62,6 +63,27 @@ type GetRange struct {
 	To   *int64
 }
 
+type GetResponse struct {
+	bytes     files.File
+	directory *directoryResponse
+}
+
+type directoryResponse struct {
+	size      uint64
```

Comment:
:thought_balloon:  Ambiguity around "size" in context of DAGs and UnixFS is super confusing, safer to be explicit about what type of size:

```suggestion
	dagSize      uint64
```